README.md

```diff
@@ -90,9 +90,11 @@ Version Control
   <img height="137em" src="https://github-profile-trophy.vercel.app/?username=Arsenic-ATG&theme=nord&no-frame=true&margin-w=4&row=1"/>
 </a>
 
+<!--
```

Comment:
didn't removed the code entirely but instead commented it out temporarily till the time the issue is not fixed